provider/go.mod

```diff
@@ -9,8 +9,8 @@ require (
 	github.com/hashicorp/terraform-provider-aws v1.60.1-0.20220923175450-ca71523cdc36
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/pulumi/providertest v0.0.2
-	github.com/pulumi/pulumi-terraform-bridge/pf v0.18.3
-	github.com/pulumi/pulumi-terraform-bridge/v3 v3.63.2
+	github.com/pulumi/pulumi-terraform-bridge/pf v0.18.4-0.20231103010134-1e955db434c6
```

Comment:
Is this SHA on master or a branch in the bridge?

Reply:
Hm, this should be on master AFAIK but I will double-check.

Reply:
Yes.